cuebot/scheduler-sim/drain_test.py

```
@@ -0,0 +1,67 @@
+
+LABEL = sys.argv[1] if len(sys.argv) > 1 else "run"
+NJOBS = int(sys.argv[2]) if len(sys.argv) > 2 else 40
+SEED  = int(sys.argv[3]) if len(sys.argv) > 3 else 9000
+PSQL = spec.psql_cmd()
+SHOW = "10000000-0000-0000-0000-000000000003"
+
+def q(sql):
+    return subprocess.run(PSQL+["-c",sql], capture_output=True, text=True, timeout=20).stdout.strip()
+
+def main():
+    chan = grpc.insecure_channel(spec.GRPC)
```

Ruff (0.15.21)

[error] Local variable `spec` referenced before assignment

(F823)